app/scripts/services/logLinks.js

```diff
@@ -65,12 +65,12 @@ angular.module('openshiftConsole')
         ")",
         "&_a=(",
           //"columns:!(_source),",
-          "columns:!(kubernetes_container_name,<%= containername %>),", 
-          "index:'<%= namespace %>.*',",
+          "columns:!(kubernetes_container_name,message),",
```

Comment:
I had mistakenly wrote 'log' in the bz. In a later filter in Fluentd we move entries from 'log' to 'message' and then we remove 'log' from the payload that is sent to Elasticsearch.  We can verify that message is correct since there isn't a 'log' column to select from the left side of the screen